### PR TITLE
Add commandLine functions, read verbose argument from command line

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

File: lib/commandLine.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  environmentVariableName?: string;
+  defaultValue?: string;
+}
+
+export function getArgument(argumentName: string, options?: GetArgumentOptions): string | undefined {
+  options = options || {};
+
+  let result: string | undefined;
+
+  let rawArgument: unknown = yargs.argv[argumentName];
```
Comment (kpajdzik) [inline lib/commandLine.ts]: Do we need yargs if we're parsing parameters manually anyway?

Comment (daschult) [reply lib/commandLine.ts]: That's a good point. Let me see if I can implement this without using yargs.

Comment (daschult) [reply lib/commandLine.ts]: Nope. `yargs` is still really useful for parsing the command line arguments. `process.argv` just provides the command line arguments as a `string[]`, whereas `yargs` turns them into a map/dictionary. I'll stick with that for now.

**Comment by daschult** [comment]
@kpajdzik Can you sign off if you're good?

**Pull request merged**